src/box/lua/config/instance_config.lua

```diff
@@ -1572,7 +1572,6 @@ return schema.new('instance_config', schema.record({
                         lua_call = schema.array({
```

Comment:
AFAIS, some of comments/code in `src/box/lua/config/applier/credentials.lua` is inconsistent with this update. What I see:

> Note that 'lua_eval', 'lua_call', 'sql' and 'universe' are special,
> they don't allow obj_name specialisation.

`lua_call` now allows it.

```lua
local singleton_object_types = {
    ['universe'] = true,
    ['lua_eval'] = true,
    ['sql'] = true,
}
if singleton_object_types[obj_type] then
    return ''
end
```

This code is from `obj_name_by_type_and_id`. I guess that it may lead to incorrect comparison of box vs config privileges and `config:reload()` will attempt to apply already given privileges.

Please, check it out.

Reply:
Comment corrected.
There were no problems with `config:reload()` since `singleton_object_types` does not contain `lua_call`.